dev/system-test/firestore.ts

```diff
@@ -1070,6 +1070,68 @@ describe('Query class', () => {
       });
   });
 
+  it('supports in', () => {
+    const testDocs = {
+      a: {zip: 98101},
+      b: {zip: 91102},
+      c: {zip: 98103},
+      d: {zip: [98101]},
+      e: {zip: ['98101', {zip: 98101}]},
+      f: {zip: {code: 500}},
+    };
+
+    return Promise.all([
+      randomCol.doc('a').set({zip: 98101}),
+      randomCol.doc('b').set({zip: 91102}),
+      randomCol.doc('c').set({zip: 98103}),
+      randomCol.doc('d').set({zip: [98101]}),
+      randomCol.doc('e').set({zip: ['98101', {zip: 98101}]}),
+      randomCol.doc('f').set({zip: {zip: 98101}}),
+    ])
+      .then(() => randomCol.where('zip', 'in', [98101, 98103]).get())
+      .then(res => {
+        expect(res.size).to.equal(2);
+        expect(res.docs.map(d => d.data())).to.deep.equal([
```

Comment:
Optional: My personal preference would be to compare the document keys, rather than the data (here and below).